Olympus2.hebontes/functions/fn_medical_createSubject.sqf

```diff
@@ -0,0 +1,57 @@
+/*
+ * Author: JoramD
+ * Spawn a subject
+ *
+ * Arguments:
+ * 0: Controller <OBJECT>
+ * 1: Stretcher <OBJECT>
+ * 2: Stretcher Name <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [controller, spawnPos, stretcher, subjectName] call TAC_Olympus_fnc_medical_createSubject
+ */
+#include "..\script_component.hpp"
+
+params ["_controller", "_spawnPos", "_stretcher", "_subjectName"];
+
+
+private _spawnSubjectAction = [
+    QGVAR(spawnSubjectAction),
+    format ["Spawn subject %1", _subjectName],
+    "",
+    {
+        (_this select 2) params ["_controller", "_stretcher", "_spawnPos"];
+
+        private _medSubjectsGroup = createGroup independent;
```

Comment:
```suggestion
        private _medSubjectsGroup = createGroup [independent, true];
```
To automatically delete when empty, otherwise you may reach maximum group count limit.

Side note, make use of pre-processor, like [here](https://github.com/Theseus-Aegis/Olympus/blob/093960e5f421b910d7f4108808a60b1e106ea4df/Olympus2.hebontes/functions/fn_initMOUT.sqf#L72), you can `#define SUBJECT_SIDE independent` on top, that gives you nice editing point if it ever wants to be switched. It's good practice to `#define` any magic numbers or enumerations like this one.